indico/modules/events/registration/templates/management/regform_list.html

```diff
@@ -53,6 +60,13 @@
                         </a>
                     </div>
                     <div class="group">
+                        <a href="{{ url_for('.manage_reglist', regform) }}"
+                           class="i-button icon-list" style="width: 183px; max-width: 183px;">
```

Comment:
Hm, probably that size only makes sense for the english text... But I don't really see a better solution.

Why a maximum width though? Wouldn't a minimum width make more sense?